From: Jeremy Bicha <jbicha@debian.org>
Date: Mon, 24 Dec 2018 00:21:02 -0500
Subject: Adjust library install directory

needs to match what /usr/bin/gjots2 is looking for

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 2ac4d23..4984de0 100755
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@ setup(
     url = "http://bhepple.freeshell.org/gjots/",
     scripts = ['bin/gjots2','bin/gjots2docbook','bin/docbook2gjots','bin/gjots2html','bin/gjots2html.py','bin/gjots2lpr','bin/gjots2emacs','bin/gjots2org','bin/org2gjots'],
     data_files = [
-        ( 'lib/gjots2',
+        ( '/usr/lib/python2.7/dist-packages/gjots2',
           [
               'lib/__init__.py', 'lib/file.py', 'lib/general.py', 'lib/prefs.py', 'lib/common.py', 'lib/find.py', 'lib/gui.py', 'lib/prefs.py', 'lib/version.py', 'lib/printDialog.py', 'lib/sortDialog.py'
           ]
